native/packages/react-native-bpk-component-text/src/common-types.js

```diff
@@ -22,13 +22,25 @@ import { type Node } from 'react';
 import PropTypes from 'prop-types';
 import { Platform, StyleSheet, type StyleObj } from 'react-native';
 
-export const TEXT_STYLES = ['xs', 'sm', 'base', 'lg', 'xl', 'xxl'];
+export const TEXT_STYLES = [
+  'caps',
+  'xs',
+  'sm',
+  'base',
+  'lg',
+  'xl',
+  'xxl',
+  'xxxl',
+];
 export type TextStyle = 'xs' | 'sm' | 'base' | 'lg' | 'xl' | 'xxl';
 
+export const EMPHASIS_STYLES = ['regular', 'semiBold', 'heavy'];
+export type EmphasisStyle = 'regular' | 'semiBold' | 'heavy';
+
 export type Props = {
   children: Node,
   textStyle: TextStyle,
-  emphasize: boolean,
+  emphasize: EmphasisStyle,
```

Comment:
I think this API might be a bit confusing. `emphasize='heavy'` doesn't make that much sense to me. @shaundon had a good idea for how we evolve this API I think it was something along the lines of deprecating this prop and adding a new one that is an enum. With this approach `emphasize` takes presedence over the new prop is provided.